2018/06-09-grid-based-movement/pawns/actor.gd

```diff
@@ -24,7 +24,8 @@ func get_input_direction():
 	)
 
 func update_look_direction(direction):
-	$Pivot/Sprite.rotation = direction.angle()
+	#$Pivot/Sprite.rotation = direction.angle()
```

Comment:
Please remove commented code

Reply:
All the requests have been resolved now. Regarding the errors, references to the pivot still existed in the Actor.tscn.